tests/filecheck/dialects/pdl/mlir-tests/infer_type_from_type_used_in_match_0.mlir

```diff
@@ -1,25 +1,25 @@
-// RUN: xdsl-opt %s --verify-diagnostics | filecheck %s
+// RUN: xdsl-opt %s | filecheck %s
 
 "builtin.module"() ({
-  "pdl.pattern"() ({
-    %0 = "pdl.type"() {constantType = i32} : () -> !pdl.type
-    %1 = "pdl.type"() : () -> !pdl.type
-    %2 = "pdl.operation"(%0, %1) {attributeValueNames = [], operand_segment_sizes = array<i32: 0, 0, 2>} : (!pdl.type, !pdl.type) -> !pdl.operation
-    "pdl.rewrite"(%2) ({
-      %3 = "pdl.operation"(%0, %1) {attributeValueNames = [], opName = "foo.op", operand_segment_sizes = array<i32: 0, 0, 2>} : (!pdl.type, !pdl.type) -> !pdl.operation
-    }) {operand_segment_sizes = array<i32: 1, 0>} : (!pdl.operation) -> ()
-  }) {benefit = 1 : i16, sym_name = "infer_type_from_type_used_in_match"} : () -> ()
+  pdl.pattern @infer_type_from_type_used_in_match : benefit(1) {
+    %0 = pdl.type : i32
+    %1 = pdl.type
+    %2 = pdl.operation -> (%0, %1 : !pdl.type, !pdl.type)
+    pdl.rewrite %2 {
+      %3 = pdl.operation "foo.op" -> (%0, %1 : !pdl.type, !pdl.type)
+    }
+  }
 }) : () -> ()
 
 
 
-// CHECK:       "builtin.module"() ({
-// CHECK-NEXT:    "pdl.pattern"() ({
-// CHECK-NEXT:      %0 = "pdl.type"() {"constantType" = i32} : () -> !pdl.type
-// CHECK-NEXT:      %1 = "pdl.type"() : () -> !pdl.type
-// CHECK-NEXT:      %2 = "pdl.operation"(%0, %1) {"attributeValueNames" = [], "operand_segment_sizes" = array<i32: 0, 0, 2>} : (!pdl.type, !pdl.type) -> !pdl.operation
-// CHECK-NEXT:      "pdl.rewrite"(%2) ({
-// CHECK-NEXT:        %3 = "pdl.operation"(%0, %1) {"attributeValueNames" = [], "opName" = "foo.op", "operand_segment_sizes" = array<i32: 0, 0, 2>} : (!pdl.type, !pdl.type) -> !pdl.operation
-// CHECK-NEXT:      }) {"operand_segment_sizes" = array<i32: 1, 0>} : (!pdl.operation) -> ()
-// CHECK-NEXT:    }) {"benefit" = 1 : i16, "sym_name" = "infer_type_from_type_used_in_match"} : () -> ()
-// CHECK-NEXT:  }) : () -> ()
+// CHECK:      builtin.module {
```

Comment:
In a lot of these you are checking all of the IR prints. Is this really necessary? I feel at least the module line is useless, right?

This leads to bigger conflict surfaces with changes in dependent parts of the project, which I feel is bad practice.

Reply:
I think the changes got conflated with changes already merged, it just needs a rebase, the diff was much smaller previously

Reply:
I removed the `builtin.module` check, yes